subjects/buzzinga/README.md

```diff
@@ -0,0 +1,53 @@
+# buzzinga
+
+### Instructions
+Write a function named `BuzZinga()` that takes a number as an argument and prints the following:
+    
+    If the number is divisible by 3, print Buz followed by a newline.
+    If the number is divisible by 5, print Zinga followed by a newline.
+    If the number is divisible by 3 and 5, print BuzZinga followed by a newline.
+    If the number is not divisible by 3 or 5, print * followed by a newline.
+
+### Expected function
+
+```go
+func BuzZinga(number int) {
+
+}
+```
+### Usage
+
+Here is a possible program to test your function:
+
+```go
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println(BuzZinga(15))
+}
+```
+And its output :
+
+```go
+$ go run . | cat -e
```

Comment:
What output is this?
If you pass 15 isn't the output just "BuzZinga" as the number is divisible by 3 and 5?

Reply:
I didn't quite understand you.

Reply:
my question is what is supposed to be the output for the number 15?
In my perspective, as it is divisible by 3 and 5 the output should be just  "BuzZinga"

Reply:
Its a bit confusing try changing the head wit this>

Write a function called `BuzZinga()` that takes a number as an argument, then loops over it and does the following for each iteration:

Reply:
Done